src/app/layout.tsx

```diff
@@ -72,13 +74,25 @@ const pendoInclude = `
               },
             });
         })("${process.env.REACT_APP_PENDO_API_KEY}");
-      }`;
+      }
+`;
 
 export default function RootLayout({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const [header, setHeader] = useState<string | undefined>();
+  const [footer, setFooter] = useState<string | undefined>();
+  useEffect(() => {
+    const fetchPartials = async () => {
+      const headerResponse = await fetch(`/api/chrome/rh-universal-nav-header`);
+      const footerResponse = await fetch(`/api/chrome/rh-unified-footer`);
```

Comment:
could you fetch the header and footer in parallel  instead of sequentially ?

Reply:
Sure thing